### PR TITLE
mdbook-emojicodes: init at 0.1.3

### PR DESCRIPTION
**Pull request by blaggacao**

### CODE REVIEW

**Comment by wegank** [comment]
`init: mdbook-emoji at v0.2.1` -> `mdbook-emoji: init at 0.2.1`

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -9026,6 +9026,8 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) CoreServices;
   };
 
+  mdbook-emoji = callPackage ../tools/text/mdbook-emoji { };
```
Comment (wegank) [inline pkgs/top-level/all-packages.nix:9029]:
```suggestion
  mdbook-emojicodes = callPackage ../tools/text/mdbook-emojicodes { };
```

Comment (blaggacao) [reply pkgs/top-level/all-packages.nix:9029]: ah! thanks. Brought to you by Overshight.

**Comment by blaggacao** [comment]
Needs [deps update](https://github.com/blyxyas/mdbook-emojicodes/pull/2) in upstream, first.

**Comment by blaggacao** [comment]
Result of `nixpkgs-review pr 214300` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>mdbook-emojicodes</li>
  </ul>
</details>

**Review by nrdxp — APPROVED** [review]

**Pull request merged**